vsintegration/src/.gitignore

```diff
@@ -0,0 +1 @@
+TestResults
```

Comment:
@latkin i tried, when i run test from visual studio test runner i have
    
    vsintegration/src/TestResults/Deployment.TE.Tests.mdf
    vsintegration/src/TestResults/Deployment.TE.Tests_log.ldf

is the wrong directory?